### DISCUSSION TITLE
I want to integrate IBM Security Verify but the GitHub repo is to old and not working.

### DISCUSSION THREAD

**Discussion by afw8k** [keycloak/keycloak#28705] [Q&A]
Hello I want to add IBM Security Verify because of Push Notifications but the https://github.com/IBM-Security/verify-keycloak-integration/tree/dependabot/maven/IBMSecurityVerifyAuthenticators/org.keycloak-keycloak-core-20.0.3 is to old... is there a way to integrate this? 

I also try to integrate with DUO but there is not up to date Repo too :(

**Comment by codespearhead** [comment]
I suggest [closing this discussion](https://docs.github.com/en/discussions/managing-discussions-for-your-community/managing-discussions#closing-a-discussion) in favor of https://github.com/keycloak/keycloak/issues/11694 (just add an emoji to the discussion) if you're just throwing another tool into the mix because of push notifications.
